psorcast/src/main/java/org/sagebionetworks/research/modules/psorcast/step/plaque_body_map/PlaqueBodyMapStepView.java

```diff
@@ -0,0 +1,79 @@
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.sagebionetworks.research.modules.psorcast.step.plaque_body_map;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.google.common.collect.ImmutableMap;
+
+import org.sagebionetworks.research.domain.step.interfaces.Step;
+import org.sagebionetworks.research.presentation.DisplayString;
+import org.sagebionetworks.research.presentation.mapper.DrawableMapper;
+import org.sagebionetworks.research.presentation.model.ColorThemeView;
+import org.sagebionetworks.research.presentation.model.ImageThemeView;
+import org.sagebionetworks.research.presentation.model.action.ActionView;
+import org.sagebionetworks.research.presentation.model.implementations.UIStepViewBase;
+
+public class PlaqueBodyMapStepView extends UIStepViewBase {
+    public static final String TYPE = "plaqueBodyMap";
+
+    public static PlaqueBodyMapStepView fromPlaqueBodyMapStep(Step step, DrawableMapper mapper) {
+        if (!(step instanceof PlaqueBodyMapStep)) {
+            throw new IllegalArgumentException("Provided step: " + step + " is not a PlaqueBodyMapStep");
+        }
+
+        UIStepViewBase activeStep = UIStepViewBase.fromUIStep(step, mapper);
+        return new PlaqueBodyMapStepView(activeStep.getIdentifier(),
+                activeStep.getActions(), activeStep.getTitle(), activeStep.getText(),
+                activeStep.getDetail(), activeStep.getFootnote(), activeStep.getColorTheme(),
+                activeStep.getImageTheme());
+    }
+
+    public PlaqueBodyMapStepView(@NonNull final String identifier,
+            @NonNull final ImmutableMap<String, ActionView> actions,
+            @Nullable final DisplayString title,
+            @Nullable final DisplayString text,
+            @Nullable final DisplayString detail,
+            @Nullable final DisplayString footnote,
+            @Nullable final ColorThemeView colorTheme,
+            @Nullable final ImageThemeView imageTheme) {
+        super(identifier, actions, title, text, detail, footnote, colorTheme, imageTheme);
+    }
+
+    @NonNull
+    @Override
+    public String getType() {

```

Comment:
@sierrawang as an experiment, you can comment out this method. This will cause this `StepView` to be rendered using the `Fragment` associated with a generic UI step (`ShowUIStepFragment`) instead of `ShowPlaqueBodyStepFragment`